relay-kafka/src/producer/mod.rs

```diff
@@ -402,7 +402,12 @@ impl Producer {
                 relay_log::error!(
                     error = &error as &dyn std::error::Error,
                     tags.variant = variant,
-                    "error sending kafka message"
+                    "error sending kafka message to topic {}",
+                    topic_name
```

Comment:
Instead of changing the name, please, add the topic name into the tags. E.g.

```rust
 relay_log::error!(
      error = &error as &dyn std::error::Error,
      tags.variant = variant,
      tags.topic = topic_name,
      "error sending kafka message"
      "error sending kafka message",
```